src/parser/actual.js

```diff
@@ -1,80 +1,113 @@
-    // given a geoid, need to find nyt equivalent "id"
-    // returns NYT-specific ID given a Tuple of county and state
-
-    let id;
+function readActual(path, obj, geoids, params) {
+    // TODO: only set up for death actual data
 
-    return id;
+    try {
+        const data = fs.readFileSync(path, 'UTF-8');
+        const lines = data.split(/\r?\n/);
 
-}
+        lines.forEach((rawLine) => {
+            const line = rawLine.split(',')
+            const geoid = line[1], date = line[2], val = parseInt(line[3]);
 
-function getActualData(geoid) {
+            // skip header and geoids we don't care about
+            if (line[0] === 'index' || !geoids.includes(geoid)) return
 
-    // const id = locateCounty(geoid);
-    const id = 1;
-    const base = 'https://coronavirus-tracker-api.herokuapp.com/v2/';
-    const path = `${base}locations?id=${id}&source=nyt&timelines=true`
+            for (let param of params) {
 
-    axios.get(path)
-    .catch(error => {
-        console.log('error', error);
-    })
-    .then(response => {
-
-        const json = response.data.locations[0]; 
-
-        const infections = json.timelines.confirmed;
-        const deaths = json.timelines.deaths;
+                obj[param][geoid].push({date, val});
+            }
+        });
+    } catch (err) {
+        console.error(`Error reading actual data`);
+    }
+}
 
-        this.props.onCountySelect([infections, deaths]);
-        this.setState({
-            'infectActual': infections,
-            'deathActual': deaths
-        })
-    });
+function aggregateActualByState(obj, states, geoids, params) {
+    // aggregates geoids by state and adds to final obj
+
+    const dates = obj['deaths']['06019'].map(i => i.date); // array of dates
+
+    // add state keys to obj
+    for (let param of params) {
+        
+        for (let state of states) {
+            let vals = [];
+            obj[param][state] = [];
+
+            // geoids for given state
+            const geoidsByState = geoids.filter(geoid => geoid.slice(0, 2) === state);
+            for (let geoid of geoidsByState) {
+
+                if (vals.length === 0) {
+                    vals = _.cloneDeep(obj[param][geoid].map(i => i.val));
+                } else {
+                    const newVals = obj[param][geoid].map(i => i.val);
+                    
+                    for (let i = 0 ; i < dates.length; i++) {
+                        vals[i] = vals[i] + newVals[i];
+                    }
+                }
+            }
+
+            for (let i = 0 ; i < dates.length; i++) {
+                const date = dates[i];
+                const val = vals[i]; 
+                obj[param][state].push({date, val})
+            }
+        }
+    }
 }
 
-countiesByGeoid()
-returnNYTcountyID()
-getActualData()
-createNYTmapping()
+module.exports = {
+    initActualObj,
+    readActual,
+    aggregateActualByState
+};
+
+// all CA and NY counties 
+const geoids = [
```

Comment:
A better way to name/separate this is to have three different const. One for CA and the other for NY, and then another const that collects the counties. The code then becomes self documenting.

Example:
```
const countyNY = [...]
const countyCA = [...]
// Pick whatever array merging method you want here.
const allCounties = [].concat(countyNY, countyCA);
```